### DISCUSSION TITLE
Maven Plugins

### DISCUSSION THREAD

**Discussion by DalekCraft2** [eccentricdevotion/TARDIS#511] [Q&A] [answered]
I am currently trying to update some of the other TARDIS plugins to make pull requests, but I believe that I can not do it until I have installed certain plugins for Eclipse and Maven. Without them, the projects seem to not recognize what `${basedir}` means in `pom.xml`. Which plugins are required to make the code be read correctly?

**Comment by Chew** [comment]
What editor are you using?

**Reply by DalekCraft2** [reply]
I am using Eclipse IDE.

**Reply by DalekCraft2** [reply]
Maybe E.D. can just send his preferences to me. Do preferences include plugin repositories?

**Comment by eccentricdevotion** [comment, marked as answer]
I use [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download), I've tried Eclipse in the past and found it horrible.
You should be able to just open the project directory in IntelliJ and it will read the pom.xml file and download what it needs.
